pilz_jog_arm_support/launch/prbt_jog_arm_sim.launch

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+
+<!--
+Copyright (c) 2019 Pilz GmbH & Co. KG
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<launch>
+  <arg name="sim" default="true" />
+
+  <!-- Control a supported gripper, e.g. 'pg70'.
+       Note: Please include the gripper model into your application URDF yourself. -->
+  <arg name="gripper" value="" />
+
+  <include file="$(find prbt_moveit_config)/launch/move_group.launch">
+    <arg name="fake_execution" value="false" />
+    <arg name="load_robot_description" value="true" />
+    <arg name="gripper" value="$(arg gripper)" />
+
+    <arg name="pipeline" value="pilz_command_planner" />
```

Comment:
with this pipeline we need depend on pilz_industrial_motion?

Reply:
`pilz_trajectory_generation` is enough. `pilz_industrial_motion` contains the python front-end, while the planner itself lives in `pilz_trajectory_generation`